cdc/entry/schema_storage.go

```diff
@@ -719,11 +723,14 @@ func (s *schemaStorageImpl) GetSnapshot(ctx context.Context, ts uint64) (*schema
 	// The infinite retry here is a temporary solution to the `ErrSchemaStorageUnresolved` caused by
 	// DDL puller lagging too much.
 	startTime := time.Now()
+	logTime := startTime
 	err := retry.Do(ctx, func() error {
 		var err error
 		snap, err = s.getSnapshot(ts)
-		if time.Since(startTime) >= 5*time.Minute && isRetryable(err) {
+		now := time.Now()
+		if now.Sub(logTime) >= 30*time.Second && isRetryable(err) {
```

Comment:
how about use `time.Since`

Reply:
`time.Since` calls `time.Now()` internally, and `time.Now()` invokes a syscall `clock_gettime`. The current approach is more performant.